### ISSUE TITLE
(Sorry idk what the title should be)This writing is invalid : <Deconstruct time="5">       <Item identifier="titanium" amount="3" />

### ISSUE THREAD

**Issue by esirprus** [Regalis11/Barotrauma#9721]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

In Vanilla you can see this 
![image](https://user-images.githubusercontent.com/44423036/185086784-72d4371e-916b-4859-8fa9-b361220a83b5.png)

seems you should get  2x plastic and 1x copper when deconstruct autoinjectorheadset, but you just get 1x plastic and 1x copper.

### Reproduction steps

1.start the game
2.input "spawnitem autoinjectorheadset inventory" on console
3.deconstruct autoinjectorheadset
4.see how much plastic you get

### Bug prevalence

Happens every time I play

### Version

0.18.15.0

### Which operating system did you encounter this bug on?

Windows

**Comment by esirprus** [comment]
No, no, what I mean is that no matter what the number is, you always get only one deconstruct product.

**Comment by JeroonBosch** [comment]
Same with body armor, seems like XML does not support amount="2" for deconstruction, which it should imo

**Comment by Regalis11** [comment]
Fixed in https://github.com/Regalis11/Barotrauma-development/commit/ee8644d21c2adfc5d80c2fdbba0932a0cef3b12b

**Comment by gerakis** [comment]
So are you going to reopen this or what?
